MoppApp/MoppApp/Signing/SivaCertViewController.swift

```diff
@@ -30,26 +30,16 @@ enum SivaAccess: String, Codable {
     case manualAccess
 }
 
-class SettingsSivaCertCell: UITableViewCell {
+class SivaCertViewController: MoppViewController {
```

Comment:
I would suggest renaming with possible refactor as currently there 2 classes with same name and similar pattern in inner and outer package:
.../Signing/SivaCertViewController: MoppViewController and .../Signing/Siva/SivaCertViewController: MoppViewController.

Reply:
For the app, there is only one "SivaCertViewController". I removed the other from repo